### DISCUSSION TITLE
【专栏夜间样式优化】建议夜间模式下专栏评论区保留浅灰底色

### DISCUSSION THREAD

**Discussion by RingingVolcano** [the1812/Bilibili-Evolved#2187] [功能建议 / Ideas]
![O{ DLM%46V4GWXSZNYLD%CE](https://user-images.githubusercontent.com/72140353/130336213-2dd28392-5a68-4736-9050-969381bb133f.png)
官方日间效果
![) {~8H5DDHL6INTY(QM%EE3](https://user-images.githubusercontent.com/72140353/130336234-7964294c-d2ae-4157-ac1f-618a13f67fc6.png)
目前Evolved的夜间

希望评论区保留那个浅灰色底衬 直接和深色背景融为一体有点割裂感

**Comment by the1812** [comment]
等b站什么时候加圆角了我就做
![image](https://user-images.githubusercontent.com/26504152/130339220-4d10aba8-c0b3-4722-8a46-1e931a8814ed.png)

**Reply by RingingVolcano** [reply]
草

**Comment by Vardariotai** [comment]
问一下大家，用夜间模式看文章区的时候，黑色文字是否正常变为白色？

**Reply by RingingVolcano** [reply]
麻烦贴出有问题的文章链接

**Reply by mokurin000** [reply]
> 麻烦贴出有问题的文章链接

https://www.bilibili.com/read/cv10468091 灰字没什么，主要是半透明图
如果能临时禁用夜间能解决这类
